eos-upgrade-eos2-to-eos3

```diff
@@ -186,8 +176,10 @@ if [ -n "$printers" ]; then
 else
     echo "No printers found"
 fi
-systemctl start cups
-systemctl start cups.socket cups.path
+
+# Restart CUPS to make extra sure that things got updated before
```

Comment:
I don't really see why this is needed because `lpstat` is implemented by communicating with CUPS. I guess it's harmless…

Reply:
It's probably not needed but I can't be 100% sure. I'm just adding it for consistency with what we had before and because it's probably harmless.